api/src/main/java/com/github/streamshub/console/api/support/serdes/ApicurioClient.java

```diff
@@ -0,0 +1,220 @@
+package com.github.streamshub.console.api.support.serdes;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+import jakarta.enterprise.inject.spi.CDI;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.github.streamshub.console.config.SchemaRegistryConfig;
+
+import io.apicurio.registry.rest.client.impl.ErrorHandler;
+import io.apicurio.rest.client.auth.Auth;
+import io.apicurio.rest.client.error.RestClientErrorHandler;
+import io.apicurio.rest.client.request.Request;
+import io.apicurio.rest.client.spi.ApicurioHttpClient;
+import io.apicurio.rest.client.util.RegistryDateDeserializer;
+import io.apicurio.rest.client.util.UriUtil;
+import io.quarkus.tls.TlsConfiguration;
+import io.quarkus.tls.TlsConfigurationRegistry;
+
+import static com.github.streamshub.console.support.StringSupport.replaceNonAlphanumeric;
+import static java.util.Objects.requireNonNull;
+
+/**
```

Comment:
Do we have a path to removing this code from the Console?

Reply:
I haven't given too much thought yet, other than to think that this kind of support should be in the Quarkus extension [1] somehow.

[1] https://github.com/quarkusio/quarkus/tree/main/extensions/schema-registry/apicurio

Reply:
Yes, I think it is worth separately opening a RFE/putting up a PR to get this code back where it belongs.  Although maybe it is worth holding off until the dust from Apicurio 3.0 has settled.

Reply:
I've opened https://github.com/quarkusio/quarkus/issues/45069 to track this.